### ISSUE TITLE
Dotenv compatibility

### ISSUE THREAD

**Issue by berga** [Droplr/aws-env#19]
hi,

this tool is simple as awesome, 
I've tested it against two dotenv libraries for node and for ruby,
but this is the result:

- ruby with [dotenv-rails](https://github.com/bkeepers/dotenv)
```
>> ENV['DB_PASSWORD']
=> "$'SecretPassword'"
```
- node with [dotenv](https://github.com/motdotla/dotenv)
```
App is starting...
key loaded: DB_USERNAME $'Username
key loaded: DB_PASSWORD $'SecretPassword
```

Can you provide two options for 
- remove export
- remove or change quoting (eg. without $ prefix)

**Comment by orfin** [comment]
Hi Berga!

Can you explain more, what is your use case - how does your entrypoint script looks like?

dotenv tools are used to export .env file content into environmental variables. With aws-env you don't need to use .env files as it is automatically exporting parameters into ENV.

**Comment by berga** [comment]
Hi Orfin,

it's not a must have for node, because we can wrap the pm2 script with aws-env bash context.
I manage several nginx+passenger installation, nginx resets all envs variables at launch so the easy way to expose environment is using dotenv library for ruby.

**Comment by orfin** [comment]
Hi @berga, I've just merged #20 that adds possibility to set output format to **dotenv**. So it should now work in your case :-)
